src/status_im/communities/core.cljs

```diff
@@ -105,6 +105,16 @@
                db
                communities)})
 
+(rf/defn handle-my-pending-requests-to-join
+  {:events [::fetched-my-communities-requests-to-join]}
+  [{:keys [db]} my-requests]
+  {:db (assoc-in db
+        [:my-pending-requests-to-join]
+        (reduce (fn [acc request]
+                  (assoc acc (aget request "communityId") (aget request "id")))
```

Comment:
these are "js" values, should I convert to keywords? or it's okay as is?

Reply:
I think we have in our guidelines that we should use the `oops/oget` macro to interop with JS. It compiles efficiently and use CLJS type hints so we reduce the surface for weird compilation behavior that I've faced already in `status-mobile` when I forgot to add `^js` in the proper place. The macro avoids some of the problems with `aget` (see https://github.com/binaryage/cljs-oops#object-operations, but they don't mention exactly which problems).

Due to resource constraints, I know that some pieces of data flow into the mobile client without being converted to cljs data structures. If you have this concern it might be okay to use oops, otherwise I usually prefer to deal with cljs data structures as early as possible to avoid surprises and polluting the code with interop macros (oops/* or native ones).

So ideally, the `my-requests` data should already come as a cljs data structure, so it's always transparent to the event handler. Doing this also helps avoid the situation of storing native js objects in the app db by mistake, which would force all consumers of the state to remember to repeatably transform them or worry about using interop macros.

Reply:
Perfeito, yes that's what I was looking for. Will update 👍